minifi-api/src/main/java/org/apache/nifi/minifi/configuration/ConfigurationChangeNotifier.java

```diff
@@ -21,6 +21,11 @@
 public interface ConfigurationChangeNotifier {
 
     /**
+     * Begins the associated notification service provided by the given implementation.  In most implementations, no action will occur until this method is invoked.
+     */
+    void initialize();
```

Comment:
This should pass a Properties object so that change notifiers can be configured. The user would add properties to the bootstrap.conf (in the same way NiFi has notification services configured there) and RunMiNiFi would pass the bootstrap properties to each Change Notifier.